lib/tedious.js

```diff
@@ -210,6 +210,7 @@ class ConnectionPool extends base.ConnectionPool {
       cfg.options.rowCollectionOnRequestCompletion = false
       cfg.options.useColumnNames = false
       cfg.options.appName = cfg.options.appName || 'node-mssql'
+      cfg.options.encrypt = this.config.encrypt != null ? this.config.encrypt : true
```

Comment:
This would be better assigned to the default options object around L200, and with strict checking on the boolean:

```js
options: Object.assign({
    encrypt: typeof this.config.encrypt === 'boolean' ? this.config.encrypt || true,
}, this.config.options);
```

Reply:
Sure, it makes sense to put the code there and improve the checking.

Reply:
However, the only reason that I stumbled over this at all is because the default value has changed from false to true in the tedious package. I got this warning: 
        deprecate('The default value for `options.encrypt` will change from `false` to `true`. Please pass `false` explicitly if you want to retain current behaviour.');
(tedious/lib/connection.js:391)
So should node-msssql also change the default, or should it be false as before?

Reply:
I'm just wondering whether our library should be being explicit or if we should be deferring to tedious and making the end user set this value explicitly... 

I'm not set on one or the other, just a thought